lighthouse-core/test/scoring-test.js

```diff
@@ -30,15 +30,6 @@ describe('ReportScoring', () => {
         {score: 0.2, weight: 1},
       ]), 0.04);
     });
-
-    it('should work for missing values', () => {
```

Comment:
yeah... :)  we assert before here that things have a score so I agree with not worrying about this case.